### DISCUSSION TITLE
Apex domain dropped on auto renewal

### DISCUSSION THREAD

**Discussion by BarryBahrami** [polymind-inc/acmebot#735] [General]
I setup this function app in Azure about 3 months ago.  The certs I issued were *.domain and .domain.  On auto renewal, it dropped the .domain and kept only the *.domain.  This caused issues for the site obviously.  Is this bug fixed by chance?  I am not sure which version is installed.  If someone can point me how to identify the version I will update this.  thank you much

**Comment by shibayan** [comment]
We are not aware of any such problems with the most recent update, and we have not changed the certificate issuance process, so we believe that it may be due to a different cause.

**Comment by BarryBahrami** [comment]
I found this was caused by another automation script in azure.  thank you for your assistance.
